### PR TITLE
Update getting-started.adoc

### PR DESCRIPTION
**Pull request by AndrejSh3**
Many people don't realize that you can run graphical apps through toolbox, and that most fedora packages can be layered onto the system without too much hassle. Maybe a part about installing external repositories might be useful here as well.

### CODE REVIEW

**Review by travier — APPROVED** [review]

File: modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -8,9 +8,8 @@ However, {variant-name} is built differently from other operating systems, and t
 These include:
 
 * Flatpak apps: this is the primary way that (GUI) apps get installed on {variant-name}.
-* Toolbox: Used primarily for CLI apps; development, debugging tools etc.
-* Package layering: The rpm-ostree tool used for host updates is a full hybrid image/package system.
-  By default the system operates in pure image mode, but package layering is useful for things like libvirt, drivers, etc.
+* Toolbox: Used primarily for CLI apps; development, debugging tools, etc, but also has support for graphical apps
```
Comment (travier) [inline modules/ROOT/pages/getting-started.adoc:11]: Nit: Missing a `.` here.

File: modules/ROOT/pages/getting-started.adoc [hunk 2]
```diff
@@ -8,9 +8,8 @@ However, {variant-name} is built differently from other operating systems, and t
...
+* Package layering: Most Fedora packages can be installed to the system with the help of package layering. By default the system operates in pure image mode, but package layering is useful for things like libvirt, drivers, etc.
```
Comment (travier) [inline modules/ROOT/pages/getting-started.adoc:12]: Let's keep the line break (+small nit):

```suggestion
* Package layering: Most Fedora packages can be installed on the system with the help of package layering.
  By default the system operates in pure image mode, but package layering is useful for things like libvirt, drivers, etc.
```

File: modules/ROOT/pages/getting-started.adoc [hunk 3]
```diff
@@ -61,10 +64,17 @@ image::sfg_libreoffice_install.png[title="LibreOffice Flatpak download"]
 Once the Flatpak is downloaded, the Software application will open a new window with an “Install” button (2).
 Click this button to begin installation.
 
+Alternatively, each application on flathub.org can be installed through the terminal by running the installation command at the bottom of the page that should look something like this:
+
+ $ flatpak install flathub <package-name>
+ 
+As an example, Spotify can be installed by running the following command which can be found on Spotify's flathub page:
```
Comment (travier) [inline modules/ROOT/pages/getting-started.adoc:71]: Let's pick a popular open source software instead.

File: modules/ROOT/pages/getting-started.adoc [hunk 4]
```diff
@@ -61,10 +64,17 @@ image::sfg_libreoffice_install.png[title="LibreOffice Flatpak download"]
...
+
+ $ flatpak install flathub com.spotify.Client
+
 === Flatpak command line
 
-In addition to using the Software app to install Flatpak apps, it is also possible to use the `flatpak` command line interface.
-See the http://docs.flatpak.org/en/latest/using-flatpak.html[Flatpak documentation] for how to do this.
+Additional details when it comes to using the flatpak command line interface can be found on the official http://docs.flatpak.org/en/latest/using-flatpak.html[Flatpak documentation].
```
Comment (travier) [inline modules/ROOT/pages/getting-started.adoc:77]:
```suggestion
Additional details about the flatpak command line interface can be found in the official http://docs.flatpak.org/en/latest/using-flatpak.html[Flatpak documentation].
```

File: modules/ROOT/pages/getting-started.adoc [hunk 5]
```diff
@@ -82,7 +92,7 @@ Most (but not all) RPM packages provided by Fedora can be installed on {variant-
 
 Currently, using package layering creates a new "deployment", or bootable filesystem root.
 It does not affect your current root.
-This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered.
+This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered or updated.
```
Comment (travier) [inline modules/ROOT/pages/getting-started.adoc:95]: Can you add a mention to `rpm-ostree install --apply-live <pkg>`? Thanks!

Comment (AndrejSh3) [reply modules/ROOT/pages/getting-started.adoc:96]: Wasn't that an experimental feature?

Comment (travier) [reply modules/ROOT/pages/getting-started.adoc:96]: This one is stable now: https://github.com/coreos/rpm-ostree/releases/tag/v2022.13

File: modules/ROOT/pages/getting-started.adoc [hunk 6]
```diff
@@ -81,9 +92,8 @@ Good examples of packages to be layered would be:
 Most (but not all) RPM packages provided by Fedora can be installed on {variant-name} using this method.
 
 Currently, using package layering creates a new "deployment", or bootable filesystem root.
-It does not affect your current root.
-This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered.
-If you don't want to reboot your system for switching to the new deployment, you can use an experimental `rpm-ostree ex apply-live` command to update the filesystem to see changes from new deployment, but it's generally expected that you use package layering sparingly, and use `flatpak` and `dnf install` inside a `toolbox` etc.
+It does not affect your current root. This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered or updated, unless you run `rpm-ostree install --apply-live <pkg>`, in which case it applies it directly to your current deployment.
```
Comment (travier) [inline modules/ROOT/pages/getting-started.adoc:95]:
```suggestion
It does not affect your current root. This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered or updated.
You can alternatively use `rpm-ostree install --apply-live <pkg>` to also temporarily apply the change directly to your currently booted deployment.
```

**Review by travier — COMMENTED** [review]
Last minor nits and we're good. Thanks for working on this.

**Pull request merged**